docs/monitor-kubernetes.md

```diff
@@ -0,0 +1,305 @@
+
+    ```yaml title="pmm-api-vmoperator.yaml"
+    apiVersion: v1
+    data:
+      api_key: <base-64-encoded-API-key>
+    kind: Secret
+    metadata:
+      name: pmm-token-vmoperator
+      #namespace: default
+    type: Opaque
+    ```
+
+4. Create the Namespace where you want to set up monitoring. The following command creates the Namespace `monitoring-system`. You can specify a different name. In the latter steps, specify this name instead of the `<namespace>` placeholder.
+    
+    ```{.bash data-prompt="$" }
+    $ kubectl create namespace monitoring-system
+    ```
+
+5. Create the Secrets object using the YAML file you created previously. Replace the `<filename>` placeholder with your value.
+
+    ```{.bash data-prompt="$" }
+    $ kubectl apply -f pmm-api-vmoperator.yaml -n <namespace>
+    ```
+
+6. Check that the secret is created. The following command checks the secret for the resource named `pmm-token-vmoperator` (as defined in the `metadata.name` option in the secrets file). If you defined another resource name, specify your value.  
+
+   ```{.bash data-prompt="$" }
+   $ kubectl get secret pmm-token-vmoperator -n <namespace>
+   ```
+
+### Create a ConfigMap to mount for kube-state-metrics 
+
+The `kube-state-metrics` (KSM) is a simple service that listens to the Kubernetes API server and generates metrics about the state of various objects - Pods, Deployments, Services and Custom Resources. 
+
+To instruct the `kube-state-metrics` what metrics to capture, create the ConfigMap and mount it to a container. 
+
+1. Edit the [example `configmap.yaml` configuration file](https://raw.githubusercontent.com/percona/percona-everest-cli/main/data/crds/victoriametrics/kube-state-metrics/configmap.yaml) and specify the `<namespace>`. The Namespace must match the Namespace where you created the Secret. 
+
+2. Apply the configuration
+
+    ```{.bash data-prompt="$" }
+    $ kubectl apply -f <path-to-configmap.yaml>
+    ```
+
+    As a result, you have the `customresource-config-ksm` ConfigMap created. 
+
+### Install the Victoria Metrics Operator Helm chart
+
+1. Add the dependency repositories of [victoria-metrics-k8s-stack](https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-k8s-stack) chart. 
+
+    ```{.bash data-prompt="$" }
+    $ helm repo add grafana https://grafana.github.io/helm-charts
+    $ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    ```
+
+2. Update the repositories.
+
+    ```{.bash data-prompt="$" }
+    $ helm repo update
+    ```
+
+3. Add the Victoria Metrics Operator repository.
+
+    ```{.bash data-prompt="$" }
+    $ helm repo add vm https://victoriametrics.github.io/helm-charts/
+    $ helm repo update
+    ```
+
+4. Export default values of `victoria-metrics-k8s-stack` chart to file `values.yaml` file.
```

Comment:
Only caveat is, the placeholders and default values file may change with new versions.
We need to check and keep updating this doc when there are new releases of helm charts are present.
We can also mention that this was taken from the chart of version X and asks the users to verify the path if different version of chart is used.

Reply:
What path?